### ISSUE TITLE
Automatically delete messages from Trash after 30 days not working

### ISSUE THREAD

**Issue by kotecky** [runbox/runbox7#1584]
The setting "Automatically delete messages from Trash after 30 days" is not working on my account. 

The setting is on: 
![image](https://github.com/runbox/runbox7/assets/2297444/9cc392b2-6aab-4cbb-9095-e0f6b6b28ff8)

The messages remain: 
![image](https://github.com/runbox/runbox7/assets/2297444/c08981df-9ec8-4327-a64d-06d31d715fbb)

There are gaps in the timeline so it could be that the routine worked somehow in the past but not fully:
![image](https://github.com/runbox/runbox7/assets/2297444/5a0c34a8-d197-4958-a436-68476a1bfdf0)

 - Browser: FF on all devices (+-4), Thunderbird on some (2), K9 on one device. 
 - Version: always the latest versions. 
 - Devices: Samsung Galaxy S23 on Android, various PCs with Win10 and Win11.

**Comment by gtandersen** [comment]
@kotecky Thanks for this report.

Were the messages deleted or moved via IMAP by any chance?

**Comment by gtandersen** [comment]
See also GL Dovecot issue #9.

**Comment by kotecky** [comment]
> @kotecky Thanks for this report.
> 
> Were the messages deleted or moved via IMAP by any chance?

Yes, i do use IMAP (Win 10/11 & Thunderbird, Android & K-9 Mail) and the web interface simultaneously.
Not sure if that explains the "gaps in the timeline" as i use the client daily (hmm, not Thunderbird, though). The gaps are present in the far past, not between the 30+ days old messages.

**Comment by kotecky** [comment]
> See also GL Dovecot issue 9.

@gtandersen The relation between the topics remains hidden to me ;-)

**Comment by gtandersen** [comment]
@kotecky Apologies for the confusion regarding the internal tracking system reference.

We are working to upgrade/replace our IMAP software which should improve this issue as well.

**Comment by kotecky** [comment]
Just to confirm that the issue is still there. Unless a special action is needed on my side? @gtandersen

**Comment by sanderbaas** [comment]
I have the exact same issue. I too use K9 mail, thunderbird and webmail at the same time. Messages are not deleted from trash automatically when deleted from Thunderbird or K-9.

**Comment by apteryks** [comment]
As discussed in https://community.runbox.com/t/cant-seem-to-empty-trash/1369/3, I also have that problem. I mostly use Gnus (in Emacs) to read and expire (delete) my emails. I also use K9 on the mobile, but more sparingly.
